pyface/wx/aui.py

```diff
@@ -0,0 +1,275 @@
+# Standard library imports.
+import logging
+
+import os
+
+# Major package imports.
+import wx
+
+# Logger.
+logger = logging.getLogger(__name__)
+
+# Multiple AUI versions are no longer supported; the C version in wx.aui is not
+# capable of supporting the windowing flexibility needed by tasks. Therefore,
+# only AGW's pure-python AUI implementation is used.
+
+from wx.lib.agw import aui
+
+# AGW's library does need some patching for some usability differences desired
+# for pyface but not for the standard wxPython version
+
+class PyfaceAuiNotebook(aui.AuiNotebook):
+    SetPageToolTip = aui.AuiNotebook.SetPageTooltip
```

Comment:
This seems to be causing some problems for running under wx < 3.  Can there perhaps be wrapped as standard methods (or even omitted entirely?).